lib/github-provider.js

```diff
@@ -1,11 +1,19 @@
 'use strict';
+const moment = require('moment');
 
 const { handleRequest: cache } = require('./request-handler');
 const {
   makeBadgeData: getBadgeData,
   makeLabel: getLabel,
-  getLogo
+  makeLogo: getLogo,
```

Comment:
sorry, I made a mistake on previous pull request. I didn't noticed  that `getLogo` is not defined in `badge-data`.  It is alias of `makeLogo`